### ISSUE TITLE
More false positives with `no-ex-assign`

### ISSUE THREAD

**Issue by blakeembrey** [buzinas/tslint-eslint-rules#118]
It's not checking for assignment, but usage by the looks of it 😦 For example, this code still fails:

```ts
try {
} catch (error) {
  if (error instanceof TSError) {}
}
```

Related to https://github.com/TypeStrong/ts-node/pull/201.

**Comment by buzinas** [comment]
Thanks for the issue. I'll check it asap. In the mean time, if you wanna take a look, I would appreciate.
